pkg/controller/controller.go

```diff
@@ -397,7 +408,6 @@ func (c *Controller) bootWithError(ctx context.Context) error {
 			For(c.newRuntimeObjectFunc()).
 			WithOptions(controller.Options{
 				MaxConcurrentReconciles: 1,
-				Reconciler:              c,
```

Comment:
Needed because latest ctrl runtime packages added validation and the reconcilier is added later on